futures-util/src/future/mod.rs

```diff
@@ -71,34 +71,6 @@ impl<T: ?Sized> FutureExt for T where T: Future {}
 /// An extension trait for `Future`s that provides a variety of convenient
 /// combinator functions.
 pub trait FutureExt: Future {
-    /// Block the current thread until this future is resolved.
-    ///
-    /// This method will consume ownership of this future, driving it to
-    /// completion via `poll` and blocking the current thread while it's waiting
-    /// for the value to become available. Once the future is resolved the
-    /// result of this future is returned.
-    ///
-    /// > **Note:** This method is not appropriate to call on event loops or
-    /// >           similar I/O situations because it will prevent the event
-    /// >           loop from making progress (this blocks the thread). This
-    /// >           method should only be called when it's guaranteed that the
-    /// >           blocking work associated with this future will be completed
-    /// >           by another thread.
-    ///
-    /// This method is only available when the `std` feature of this
-    /// library is activated, and it is activated by default.
-    ///
-    /// # Panics
-    ///
-    /// This function does not attempt to catch panics. If the `poll` function
-    /// of this future panics, panics will be propagated to the caller.
-    #[cfg(feature = "std")]
-    fn wait(self) -> result::Result<Self::Item, Self::Error>
```

Comment:
:cry:

Reply:
Do you think this should stick around for now? (not sure if we're ready to remove?)

Reply:
We can always add it back in some form or another. Up to you how you want to handle it vs. the churn in the tests.